src/Queue.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Equip\Queue;
+
+use Equip\Queue\Driver\DriverInterface;
+
+class Queue
+{
+    const DEFAULT_QUEUE = 'queue';
+    
+    /**
+     * @var DriverInterface
+     */
+    private $driver;
+    
+    /**
+     * @param DriverInterface $driver
+     */
+    public function __construct(DriverInterface $driver)
+    {
+        $this->driver = $driver;
+    }
+
+    /**
+     * Add message to queue
+     * 
+     * @param string $name
+     * @param array $data
+     * @param string $queue
+     *
+     * @return bool
+     */
+    public function add($name, array $data = [], $queue = self::DEFAULT_QUEUE)
```

Comment:
Should I add some sort of `MessageEnvelope` abstract/interface so other developers are able to create their own sort of message/serializer?